apps/coscrad-frontend/src/components/resource-info/presenters/resource-info.presenter.tsx

```diff
@@ -15,16 +16,25 @@ IAggregateInfo<ResourceType> & { route: string }): JSX.Element => (
     <Link to={`/${route}`}>
         <Card>
             {/* TODO Handle pluralization properly as soon as we have a Resource Type whose plural form is irregular */}
```

Comment:
This is a recurring theme. Do we have a story for this yet?

Reply:
Yes, here it is: https://www.pivotaltracker.com/story/show/184310526